### PR TITLE
Add State component which allow to use different colors depends on the condition. Add new menu bar

### PR DESCRIPTION
**Pull request by volodymyrkochyn**
Updates:
* Add color indication for table components
* Add custom menu bar
* Code improvements

image

image

Notes:
* As for now menus does not implemented
* Conditions for different colors are dummy and will be defined later

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "author": "Rapid Silicon",
   "main": "main.js",
   "scripts": {
-    "start": "webpack --mode development && electron .",
+    "start": "webpack --mode development && NODE_ENV=development electron .",
```
Comment (ravikiranchollangi) [inline package.json:8]: Is the dev mode temporary?

File: src/components/DMAComponent.js [hunk 2]
```diff
@@ -42,17 +43,22 @@ function DMAComponent({ device }) {
     'Channel 1', 'Channel 2', 'Channel 3', 'Channel 4',
   ];
 
+  const warn = 0.001; // TBD
```
Comment (ravikiranchollangi) [inline src/components/DMAComponent.js:46]: What are these tbd ?

**Review by ravikiranchollangi — APPROVED** [review]
Couple of comments

File: package.json (hunk 1 above)
Comment (volodymyrkochyn) [reply package.json:8]: dev mode is for daily development. For production it must be something else.

File: src/components/DMAComponent.js (hunk 2 above)
Comment (volodymyrkochyn) [reply src/components/DMAComponent.js:46]: To be defined... As for now I don't know which values I can use for yellow and red colors

**Pull request merged**